.azure/scale.csv

```diff
@@ -0,0 +1,26 @@
+stage,siteId,servers,endingAddress,authenticationMethod,dcPort,rbacAdminGroupObjectIds,country,streetAddress1,enableProvisioners,managementAdapters,serverPorts,defaultGateway,startingAddress,rdmaEnabled,aksArc-lnet-dnsServers,stateOrProvince,contactName,postalCode,domainServerIP,storageConnectivitySwitchless,aksArc-controlPlaneIp,aksArc-lnet-startingAddress,emailList,dnsServers,aksArc-lnet-addressPrefix,mobile,storageNetworks,virtualHostIp,location,domainFqdn,city,phone,aksArc-lnet-endingAddress,companyName,adouSuffix,aksArc-lnet-defaultGateway
```

Comment:
## Arc IaC Automation generated
Next steps: 
  - [ ] Setup local git hook by running command in the root folder of cloned repo: git config --local core.hooksPath ./.azure/hooks/  
  - [ ] Fill the csv file and commit, the struct of the csv file can be found on [here](https://github.com/Azure/Edge-infrastructure-quickstart-template/blob/main/doc/Scale-with-automation.md#methodology)
  - [ ] Wait for the scale workflow run, and then resolve this comment and merge the pull request

 If you have any problem, please file an [issue](https://github.com/Azure/Edge-infrastructure-quickstart-template/issues/new/choose) or contact arcIaCSupport@microsoft.com